sql/insert.go

```diff
@@ -88,6 +95,22 @@ func (p *planner) Insert(n *parser.Insert) (planNode, error) {
 				b.Put(key, string(t))
```

Comment:
This would work for the primary key, but not for secondary keys, since the uniqueness constraint applies to a prefix of the proper key.